R/pkg/R/schema.R

```diff
@@ -68,6 +68,28 @@ structType.structField <- function(x, ...) {
   structType(stObj)
 }
 
+#' Create \code{structType} for a given DDL-formatted string.
+#'
+#' Create \code{structType} for a given DDL-formatted string, which is a comma separated list
+#' of field definitions, e.g., a INT, b STRING.
+#'
+#' @param x The data schema defined in a DDL-formatted string.
+#' @examples
+#'\dontrun{
+#' structTypeFromDDL("a INT, b STRING")
+#' }
+#' @note structTypeFromDDL since 2.3.0
+structTypeFromDDL <- function(x) {
+  if (!is.character(x)) {
+    stop("schema name must be a DDL-formatted string.")
```

Comment:
minor nit for clarity: `schema must be a DDL-formatted string.`
btw, is it possible to support JSON format string in R too?